rest/src/commonMain/kotlin/builder/interaction/OptionsBuilder.kt

```diff
@@ -96,7 +96,6 @@ public sealed class BaseChoiceBuilder<T>(
  *
  * @see LocalizedNameCreateBuilder
  */
-@KordDsl
```

Comment:
why was this removed?

Reply:
Probably because it's not a request builder, it shouldn't really mater as it's just a superclass for request builders, but I'll add it again